test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -430,14 +434,15 @@ UCS_TEST_P(test_ucp_sockaddr, listen_inaddr_any) {
 
     struct sockaddr_in inaddr_any_listen_addr;
 
-    inaddr_any_addr(&inaddr_any_listen_addr, test_addr.sin_port);
+    inaddr_any_addr(&inaddr_any_listen_addr, test_addr.get_port());
 
     UCS_TEST_MESSAGE << "Testing "
                      << ucs::sockaddr_to_str(
                         (const struct sockaddr*)&inaddr_any_listen_addr);
 
-    start_listener(cb_type(), (const struct sockaddr*)&inaddr_any_listen_addr);
-    connect_and_send_recv((const struct sockaddr*)&test_addr, false);
+    start_listener(cb_type(), (const struct sockaddr*)&inaddr_any_listen_addr,
```

Comment:
how does this test work with ipv6?

Reply:
this one tests inaddr_any only for ipv4. 
inaddr_any for ipv6 is tested in uct as one of the resources - https://github.com/openucx/ucx/blob/master/test/gtest/uct/uct_test.cc#L217 
should i add testing in ucp for inaddr_any for ipv6 as well?

Reply:
what if there is no ipv4 address? or where is it checked?

Reply:
looks like it also failed in jenkins http://hpc-master.lab.mtl.com:8080/blue/rest/organizations/jenkins/pipelines/ucx/runs/2161/nodes/108/steps/1626/log/?start=0:
```
[2019-10-24T15:51:18.626Z] [ RUN      ] shm/test_ucp_sockaddr.listen_inaddr_any/1
[2019-10-24T15:51:18.626Z] [     INFO ] Testing 0.0.0.0:49224
[2019-10-24T16:06:25.326Z] /scrap/jenkins/workspace/ucx-4/contrib/../test/gtest/common/test_helpers.cc:49: Failure
[2019-10-24T16:06:25.326Z] Failed
[2019-10-24T16:06:25.326Z] Connection timed out - abort testing
[2019-10-24T16:06:25.326Z] [boo31:11025:0:11025] Caught signal 6 (Aborted: tkill(2) or tgkill(2))
[2019-10-24T16:06:25.326Z] ==== backtrace (tid:  11025) ====
[2019-10-24T16:06:25.326Z]  0 0x0000003479c0c465 pthread_spin_lock()  ???:0
[2019-10-24T16:06:25.326Z] =================================
[2019-10-24T16:06:25.326Z] Sending notification to alinas@mellanox.com
[2019-10-24T16:06:25.326Z] [boo31:11025:0:11025] Process frozen...
[2019-10-24T18:29:35.292Z] make: *** [test] Terminated
script returned exit code 124
```